### PR TITLE
fix service tag

### PR DESCRIPTION
**Pull request by swaroopar**

### CODE REVIEW

File: modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public List<CredentialDefinition> getCredentialDefinitionsByCsp(
      * @param type The type of credential.
      * @return Link of credential openApi url.
      */
-    @Tag(name = "Services Available",
+    @Tag(name = "Credentials Management",
             description = "APIs to query the available services.")
```
Comment (baixinsui) [inline modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java:129]: Please update the description.

File: modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java [hunk 2]
```diff
@@ -327,7 +327,7 @@ public Boolean deleteCredentialByServiceId(
...
```
Comment (baixinsui) [inline modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java:331]: Please update the description.

File: modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java (hunk 1 above)
Comment (swaroopar) [reply modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java]: fixed

File: modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java (hunk 2 above)
Comment (swaroopar) [reply modules/api/src/main/java/org/eclipse/xpanse/api/CredentialManageApi.java]: fixed

**Comment by swaroopar** [comment]
@baixinsui comments are fixed.

**Review by baixinsui — APPROVED** [review]
LGTM.

**Review by iskey — APPROVED** [review]
looks good.

**Pull request merged**